instance/tests/models/test_openedx_instance.py

```diff
@@ -1220,15 +1225,12 @@ def _verify_vm_dns_records(self, domain, active_vm_ips):
         Helper function to verify a set of DNS records for active VMs.
         """
         dns_ips = set()
-        vm_indices = set()
         for record in gandi.api.list_records(domain):
             match = re.match(r'vm(\d+)\.', record['name'])
             if match:
                 self.assertEqual(record['type'], 'A')
                 dns_ips.add(record['value'])
-                vm_indices.add(int(match.group(1)))
         self.assertEqual(dns_ips, set(active_vm_ips))
-        self.assertEqual(vm_indices, set(range(1, len(vm_indices) + 1)))
```

Comment:
@pkulkark, is there a reason for which you removed these lines? These are just testing the indices used in the active appserver records and are useful.

Reply:
This was causing the test case to fail earlier (mainly because of the way they were checked) but works now after I changed the logic. Reverted the change.